tests/st/runtime/test_assemble.py

```
@@ -257,45 +238,52 @@ def compute_expected(self, tensors, params=None):
         """Vec→Vec single loop + pl.slice: dynamic row gather into left half."""
-        result = test_runner.run(TileAssembleRowByRowTestCase())
+        result = test_runner.run(TileAssembleRowByRowTestCase(platform=platform))
         assert result.passed, f"Test failed: {result.error}"
 
     @pytest.mark.skip(
         reason="Sim bug: Vec→Vec assemble with pl.slice produces wrong output (496/1024 mismatch)"
     )
-    def test_tile_assemble_double_loop(self, test_runner):
+    @pytest.mark.parametrize("platform", PLATFORMS)
+    def test_tile_assemble_double_loop(self, test_runner, platform):
         """Vec→Vec nested loops + pl.slice: batch×head two-level index (b*8+i)."""
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)